internal/host/validator.go

```diff
@@ -615,3 +627,25 @@ func isInvalidImageStatus(imageStatus *models.ContainerImageAvailability) bool {
 	return imageStatus.Result == models.ContainerImageAvailabilityResultFailure ||
 		(imageStatus.SizeBytes > 0 && imageStatus.DownloadRate < ImageStatusDownloadRateThreshold)
 }
+
+func (v *validator) sufficientDiskSpeed(c *validationContext) ValidationStatus {
+	info, err := v.getBootDeviceInfo(c.host)
+	if err != nil {
+		return ValidationError
+	}
+
+	return boolValue(info == nil || info.DiskSpeed == nil || !info.DiskSpeed.Tested || info.DiskSpeed.ExitCode == 0)
```

Comment:
if no info then there is sufficient disk speed?

Reply:
Yes.  Since we don't have results yet.  The test is being run during the preparation for install.  So for failure, a previous run has to fail.

Reply:
I don't understand how if there are no results it should be marked as sufficient

Reply:
The validation is performed during the preparing for installation stage.  The reason for that is that it is intrusive validation since it writes the disk.   So, in order to start installation - all validations should be marked as successful.  This is also the convention that the UI is expecting.
So in order for this validation to succeed one of the following conditions must be true:
- The validation has not been executed yet.
- The validation has been run during a previous installation attempt, and was successful.